wiki/Test-Rules.md

```diff
@@ -1,657 +1,258 @@
-   "HistoryExtractionRules":"",
-   "ProcessRules":null,
-   "ProcessFunction":null
-}
-{
-   "TemplateName":"ARTIFACT_1",
-   "Type":"EiffelConfidenceLevelModifiedEvent",
-   "TypeRule":"meta.type",
-   "IdRule":"meta.id",
-   "StartEvent":"NO",
-   "IdentifyRules":"links | [?type=='SUBJECT'].target",
-   "MatchIdRules":{
-      "_id":"%IdentifyRules_objid%"
-   },
-   "ExtractionRules":"{confidenceLevels :[{ eventId:meta.id, time:meta.time, name:data.name, value:data.value}]}",
-   "ArrayMergeOptions":"",
-   "HistoryIdentifyRules":"",
-   "HistoryExtractionRules":"",
-   "ProcessRules":null,
-   "ProcessFunction":null
-}
-{
-   "TemplateName":"ARTIFACT_1",
-   "Type":"EiffelArtifactCreatedEvent",
-   "TypeRule":"meta.type",
-   "IdRule":"meta.id",
-   "StartEvent":"YES",
-   "IdentifyRules":"[meta.id]",
-   "MatchIdRules":{
-      "_id":"%IdentifyRules_objid%"
-   },
-   "ExtractionRules":"{ id : meta.id, type : meta.type, time : meta.time, gav : data.gav, fileInformation : data.fileInformation, buildCommand : data.buildCommand }",
-   "DownstreamIdentifyRules":"links | [?type=='COMPOSITION'].target",
-   "DownstreamMergeRules":"{\"externalComposition\":{\"eventId\":%IdentifyRules%}}",
-   "DownstreamExtractionRules":"{artifacts: [{id : meta.id}]}",
-   "ArrayMergeOptions":"",
-   "HistoryIdentifyRules":"links | [?type=='COMPOSITION'].target",
-   "HistoryExtractionRules":"{internalComposition:{artifacts: [{id : meta.id}]}}",
-   "HistoryPathRules":"{artifacts: {id: meta.id}}",
-   "ProcessRules":null,
-   "ProcessFunction":null
-}
-{
-   "TemplateName":"ARTIFACT_1",
-   "Type":"EiffelTestCaseFinishedEvent",
-   "TypeRule":"meta.type",
-   "IdRule":"meta.id",
-   "StartEvent":"NO",
-   "IdentifyRules":"links | [?type=='TEST_CASE_EXECUTION'].target",
-   "MatchIdRules":{
-      "$and":[
-         {
-            "aggregatedObject.testCaseExecutions.testCaseStartedEventId":"%IdentifyRules%"
-         }
-      ]
-   },
-   "ExtractionRules":"{ testCaseFinishEventId:meta.id, testCaseFinishedTime:meta.time, testCase:data.outcome}",
-   "MergeResolverRules":"{\"testCaseStartedEventId\":%IdentifyRules%}",
-   "ArrayMergeOptions":"",
-   "HistoryIdentifyRules":"",
-   "HistoryExtractionRules":"",
-   "ProcessRules":"{testCaseDuration : diff(testCaseExecutions[0].testCaseFinishedTime, testCaseExecutions[0].testCaseStartedTime)}",
-   "ProcessFunction":"difference"
-}
-</pre>
+<h3>Find Aggregated Object</h3>
+
+<p>Clicking on "Find Aggregated Object" button will start the aggregation process. If rules and events are correct, a pop-up window with the aggregated object will show up on the screen.</p>
+
+<img src="./images/GUI_TestRules_Aggregated_Object.png"></img>
 
-<p>Also note that if you refer to a key that does not exist, a value of null (or the language equivalent of null) is
-    returned.</p>
+**_More information about how to write rules can be found [here](https://github.com/eiffel-community/eiffel-intelligence)._**
```

Comment:
This links to the ei back end repo. Is there a more specific link? I found wiki docs Rules.md
 and TestRulesUserGuide.md, the latter which I'm not sure if it's supposed to be there.

Reply:
I could change this link to the more specific link for example to Rules.md page. I though that in main page has better overview of all rules sections, but is maybe better to link to one specific.

Reply:
Yes, a link to the rules documentation in backend is better that this general link.

Reply:
Done